pytestqt/qtbot.py

```diff
@@ -495,8 +560,19 @@ def result(*args, **kwargs):
                 setattr(cls, method_name, method)
 
 
-# provide easy access to SignalTimeoutError to qtbot fixtures
+class TimeoutError(Exception):
```

Comment:
What about inheriting from `SignalTimeoutError` (for backwards compatibility) and deprecating `SignalTimeoutError`, like discussed in https://github.com/pytest-dev/pytest-qt/pull/157#issuecomment-241227483 some while ago?